docs/squadcast.md

```diff
@@ -38,39 +38,53 @@ terraformer import squadcast --resources=team --region=us --team-name="Default T
 ### Flags:
 
 - `--team-name`
-
   - Required for the following resources:
-    - squad
-    - service
+    - deduplication_rules
     - escalation_policy
+    - routing_rules
+    - runbook
+    - schedules
+    - service
+    - slo
+    - squad
+    - suppression_rules
+    - tagging_rules
     - team_member
     - team_roles
-    - slo
-    - runbook
 
-- `--region`
+- `--service-name`
+  - Required for the following resources:
+    - deduplication_rules
+    - routing_rules
+    - suppression_rules
+    - tagging_rules
+
+- `--schedule-name`
+  - Required for the following resources:
+    - schedule
 
+- `--region`
   - Supported Values:
     - `us`
     - `eu`
 
+- `--refresh-token` (optional)
+  - Supported Values:
+    - <YOUR_SQUADCAST_REFRESH_TOKEN>
+
 ### Supported resources:
 
-- `user`
-  - [squadcast_user](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/user)
-- `team`
-  - [squadcast_team](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/team)
-- `team_member`
-  - [squadcast_team_member](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/team_member)
-- `team_roles`
-  - [squadcast_team_roles](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/team_roles)
-- `squad`
-  - [squadcast_squad](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/squad)
-- `service`
-  - [squadcast_service](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/service)
-- `escalation_policy`
-  - [squadcast_escalation_policy](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/escalation_policy)
-- `runbook`
-  - [squadcast_runbook](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/runbook)
-- `slo`
-  - [squadcast_slo](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/slo)
+- [`deduplication_rules`](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/deduplication_rules)
+- [`escalation_policy`](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/escalation_policy)
+- [`routing_rules`](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/routing_rules)
+- [`runbook`](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/runbook)
+- [`schedule`](https://registry.terraform.io/providers/SquadcastHub/squadcast/latest/docs/resources/schedule)
```

Comment:
remove schedule resource reference from everywhere like we discussed in slack

Reply:
I have removed the reference from the doc but left the code intact, since dheeraj mentioned backward compatibility should I remove the entire code or just remove it from the supported resources list in `squadcast_provider.go` file and leave the schedule code as it is?

Reply:
sure